sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_asset_utils.py

```diff
@@ -652,6 +652,32 @@ def _create_or_update_autoincrement(
     return result
 
 
+def _get_next_version_from_container(
+    name: str,
+    container_operation: Any,
+    resource_group_name: str,
+    workspace_name: str,
+    registry_name: str,
+    **kwargs,
+) -> str:
+    try:
+        container = container_operation.get(
+            name=name,
+            resource_group_name=resource_group_name,
+            registry_name=registry_name,
+            **kwargs,
+        ) if registry_name else container_operation.get(
+            name=name,
+            resource_group_name=resource_group_name,
+            workspace_name=workspace_name,
+            **kwargs,
+        )
+        version = container.properties.next_version
```

Comment:
Sanity question: The service "next version" changes are rolled out all regions? With so much CCOA I want to make sure we don't accidentally release faster than service.

Reply:
I checked with couple of prod regions, and believe this is avaiable, but let me confirm with Edwards. Also this will not be released before January...